packages/client/wallets/aa/front/package.json

```diff
@@ -9,8 +9,8 @@
         "lint": "next lint"
     },
     "dependencies": {
-        "@crossmint/client-sdk-aa": "0.2.1",
-        "@crossmint/common-sdk-base": "0.0.6",
+        "@crossmint/client-sdk-aa": "file: ../../../",
+        "@crossmint/common-sdk-base": "file: ../../../../../../common/base",
```

Comment:
we'll have to switch this back–do you need any SDK releases?

Reply:
The front demo was done to test the changes locally before push them. The problem rn is that the frontend demo is not compatible with v.5.0.0.

I'll switch this back to have it working so we can merge it, but we need to keep it in mind.